packages/mobile/fastlane/Fastfile

```diff
@@ -89,9 +89,9 @@ platform :ios do
   desc 'Prepare TestFlight'
   lane :prepare_testflight_and_upload do |options|
     api_key = app_store_connect_api_key(
-      key_id: '3NK3HR2Q69',
+      key_id: '3B5PGG27Q4',
       issuer_id: 'dcfc5c64-36b5-46a2-af36-84acc93d2b62',
-      key_filepath: '/Users/runner/work/release-bundler/release-bundler/certificates/AuthKey_3NK3HR2Q69.p8',
+      key_filepath: '/Users/runner/work/release-automation/release-automation/temp/AuthKey.p8',
```

Comment:
Instead of hardcoding this, would be great to use an environment variable.
This way it's also easier to run manually, same for `key_id`.

Reply:
Partially fulfilled here: https://github.com/valora-inc/release-automation/pull/14/files#r710047135. Will make the path an environment variable as well!